src/data/polynomial/basic.lean

```diff
@@ -101,6 +101,9 @@ by { dsimp [monomial, coeff], rw finsupp.single_apply, congr }
 
 lemma coeff_X : coeff (X : polynomial R) n = if 1 = n then 1 else 0 := coeff_monomial
 
+lemma coeff_X_ne_one {n : ℕ} (hn : n ≠ 1) : coeff (X : polynomial R) n = 0 :=
```

Comment:
Would `of_ne_one` be a better name? Your name implies to me that this is `coeff x n ≠ 1`

Reply:
```suggestion
lemma coeff_X_of_ne_one {n : ℕ} (hn : n ≠ 1) : coeff (X : polynomial R) n = 0 :=
```